src/draw/vg_lite/lv_vg_lite_utils.c

```diff
@@ -1028,13 +1028,18 @@ void lv_vg_lite_disable_scissor(void)
     LV_VG_LITE_CHECK_ERROR(vg_lite_disable_scissor());
 }
 
+int flush_cnt;
+
 void lv_vg_lite_flush(struct _lv_draw_vg_lite_unit_t * u)
 {
     LV_ASSERT_NULL(u);
     LV_PROFILER_BEGIN;
 
     u->flush_count++;
 
+    LV_LOG_USER("++ flush_count = %d", u->flush_count);
```

Comment:
I assume this is also a debugging left-over.
If needed, then please change to TRACE and simple text:
```suggestion
    LV_LOG_TRACE("flush_count = %d", u->flush_count);
```